resources/dev-config.edn

```diff
@@ -0,0 +1,2 @@
+{:username "chaselambert.dev"
+ :password #env BSKY_APP_PASSWORD}
```

Comment:
Yes, I use `[aero](https://github.com/juxt/aero)` for such things. I will remove for now as we explore the read only (hence no auth needed) aspects as discussed. I'll also put this file in the `.gitignore` (which I think I intended to do originally) in case we come back to it.